docs/content/preview/yugabyte-platform/manage-deployments/remove-nodes.md

```diff
@@ -36,9 +78,27 @@ A node status displayed in the UI is not always entirely indicative of the node'
 
 {{< /note >}}
 
+## Start and stop node processes
+
+### Stop a process
+
+If a node needs to be briefly taken out of service (for example, to perform a quick OS patch), you can click its associated **Actions > Stop Processes**. It is expected that this node will be returned to service soon through the **Actions > Start Processes** operation.
+
+After the YB-TServer and (where applicable) YB-Master server are stopped, the node status is updated and the instance is ready for the planned system changes.
+
+Generally, when a YB-Master is stopped on a node, YugabyteDB Anywhere automatically attempts to start a new YB-Master on another node in the same Availability Zone as the node on which YB-Master is stopped. This ensures that the number of YB-Master servers equals the replication factor (RF) and YB-Master servers are never underreplicated.
+
+It is recommended not to stop more than (RF - 1) / 2 processes at any given time. For example, on an RF=3 cluster with three nodes, there can only be one node with stopped processes to allow the majority of the nodes to perform Raft consensus operations.
```

Comment:
```suggestion
In general, it is not recommended to execute the "Stop Processes" action on multiple nodes simultaneously. For example, two stopped nodes might share a common tablet. This could cause unavailability on a universe with replication factor 3.
```